pkg/virtctl/guestfs/guestfs_test.go

```diff
@@ -0,0 +1,146 @@
+package guestfs_test
+
+import (
+	"fmt"
+
+	"github.com/golang/mock/gomock"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/testing"
+
+	kubevirtv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/pkg/virtctl/guestfs"
+	"kubevirt.io/kubevirt/tests"
+)
+
+const (
+	commandName   = "guestfs"
+	pvcName       = "test-pvc"
+	testNamespace = "default"
+)
+
+func fakeAttacherCreator(client *guestfs.K8sClient, p *corev1.Pod, command string) error {
+	return nil
+}
+
+var _ = Describe("Guestfs shell", func() {
```

Comment:
These tests should be assigned to a SIG by adding a `[sig-<sig-name>]` to the description, being `<sig-name>` `compute`, `storage`, `network` or `operator`. There are sigs like network and storage that move their tests under specific directories and use the `SIGDescribe` function like https://github.com/kubevirt/kubevirt/blob/master/tests/storage/datavolume.go#L57

Reply:
@fgimenez is that true also for unit tests? The `imageupload` command doesn't have any label:https://github.com/kubevirt/kubevirt/blob/master/pkg/virtctl/imageupload/imageupload_test.go#L47

Reply:
@alicefr nope you are right, we need the sig labels only in functional tests. Sorry I linked the wrong line, where we should add the label is here https://github.com/kubevirt/kubevirt/pull/5402/files#diff-ef4152e5c3243da61c8b877c0bc1893738e8d080cf17904b8f5b1c74d96e1e05R36